app/helpers/application_helper.rb

```diff
@@ -437,17 +437,14 @@ def replace_plugin_html(name)
 
   def theme_lookup(name)
     Theme.lookup_field(theme_ids, mobile_view? ? :mobile : :desktop, name)
-      &.html_safe
```

Comment:
This comment removes the automat `html_safe` calls regardless of what is being looked up, but the commit seems to focus on description only. Does this mean it's possible other fields that were previously escaped would no longer be?